.github/workflows/staging-copy-artifacts.yml

```diff
@@ -2,7 +2,7 @@ name: Staging Copy Artifacts
 
 on:
 #  schedule:
-#    - cron: '30 10 * * *' 
+#    - cron: '0 9 * * *' 
```

Comment:
Why change the time from 10:30 UTC to 9:00 UTC?

Reply:
As the building workflows are set to start at 10. So I moved RC to 9 UTC.